src/client/common/cy/enrichment-stylesheet.js

```diff
@@ -1,6 +1,15 @@
 const cytoscape = require('cytoscape');
 
-const iStylesheet=cytoscape.stylesheet()
+function getNodeSize( geneCount ){
+  let size = ( geneCount <= 1000 ) ? mapGeneCountToSize(geneCount) : mapGeneCountToSize(1000);
```

Comment:
It would be easier to understand this code if you did something like:
```js
return mapGeneCountToSize(Math.min( geneCount, 1000 ));
```

Reply:
That is a lot better, thank you!